/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/torchvision/transforms/transforms.py:890: UserWarning: Argument interpolation should be of type InterpolationMode instead of int. Please, use InterpolationMode enum.
  warnings.warn(
/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/torch/utils/data/dataloader.py:487: UserWarning: This DataLoader will create 10 worker processes in total. Our suggested max number of worker in current system is 8 (`cpuset` is not taken into account), which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
submitit ERROR (2022-05-01 22:50:14,154) - Submitted job triggered an exception
Traceback (most recent call last):
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/submitit/core/_submit.py", line 11, in <module>
    submitit_main()
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/submitit/core/submission.py", line 72, in submitit_main
    process_job(args.folder)
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/submitit/core/submission.py", line 65, in process_job
    raise error
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/submitit/core/submission.py", line 54, in process_job
    result = delayed.result()
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/submitit/core/utils.py", line 133, in result
    self._result = self.function(*self.args, **self.kwargs)
  File "submitit_pretrain.py", line 62, in __call__
    trainer.main(self.args)
  File "/Users/dongheelee/Desktop/GitHub/mae/main_pretrain.py", line 158, in main
    model.to(device)
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/torch/nn/modules/module.py", line 907, in to
    return self._apply(convert)
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/torch/nn/modules/module.py", line 578, in _apply
    module._apply(fn)
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/torch/nn/modules/module.py", line 578, in _apply
    module._apply(fn)
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/torch/nn/modules/module.py", line 601, in _apply
    param_applied = fn(param)
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/torch/nn/modules/module.py", line 905, in convert
    return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
  File "/Users/dongheelee/Library/Python/3.8/lib/python/site-packages/torch/cuda/__init__.py", line 210, in _lazy_init
    raise AssertionError("Torch not compiled with CUDA enabled")
AssertionError: Torch not compiled with CUDA enabled
